test/domain/reaction/remove.spec.ts

```diff
@@ -0,0 +1,38 @@
+
+import { describe, expect, it } from 'vitest';
+import { COMIC_DATA_URL, COMIC_RECORD_TYPE, COMMENT_MESSAGE, COMMENT_RECORD_TYPE, EXISTING_REQUESTER, POST_ID, REACTION_RECORD_TYPE, createComicReaction, createCommentReaction, createDatabase, createFileStorage, remove } from './_fixtures/reaction.fixture';
+
+describe('domain/reaction/remove', () =>
+{
+    it('should soft delete a reaction with a comment', async () =>
+    {
+        const database = await createDatabase();
+        const reaction = await createCommentReaction(EXISTING_REQUESTER, POST_ID, COMMENT_MESSAGE);
```

Comment:
Why is the comment created here? We commonly prepare the complete database in the fixture so we keep our tests clean and focused.